### PR TITLE
[REVIEW] Normalize NANs and Zeroes (JNI bindings) [skip ci]

### PR DESCRIPTION
**Pull request by mythrocks**
#3291 added `cudf::normalize_nans_and_zeros`.
This commit adds the JNI bindings, for use from Spark.

### CODE REVIEW

File: java/src/main/java/ai/rapids/cudf/ColumnVector.java [hunk 1]
```diff
@@ -690,6 +690,15 @@ public static ColumnVector concatenate(ColumnVector... columns) {
     }
   }
 
+  private static native long normalizeNANsAndZeros(long viewHandle) throws CudfException;
+
+  public ColumnVector normalizeNANsAndZeros() {
```
Comment (revans2) [inline java/src/main/java/ai/rapids/cudf/ColumnVector.java]: Please add in some javadocs to explain what this is doing.

File: java/src/main/java/ai/rapids/cudf/ColumnVector.java [hunk 2]
```diff
@@ -690,6 +690,15 @@ public static ColumnVector concatenate(ColumnVector... columns) {
     }
   }
 
+  private static native long normalizeNANsAndZeros(long viewHandle) throws CudfException;
```
Comment (revans2) [inline java/src/main/java/ai/rapids/cudf/ColumnVector.java]: nit: we have been placing the native declarations in a single group, but not very consistently.

Comment (mythrocks) [reply java/src/main/java/ai/rapids/cudf/ColumnVector.java]: I thought that too. I wasn't sure where to place it. I'll move this to the end of the largest `native` block.

File: java/src/main/java/ai/rapids/cudf/ColumnVector.java (hunk 1 above)
Comment (mythrocks) [reply java/src/main/java/ai/rapids/cudf/ColumnVector.java]: Righto. Will do shortly.

Comment (mythrocks) [reply java/src/main/java/ai/rapids/cudf/ColumnVector.java]: Updated docs.

File: java/src/main/java/ai/rapids/cudf/ColumnVector.java (hunk 2 above)
Comment (mythrocks) [reply java/src/main/java/ai/rapids/cudf/ColumnVector.java]: Moved native method to more appropriate location.

**Comment by mythrocks** [comment]
w007! Thanks, @revans2 (&& @kkraus14).

**Pull request merged**